pkgs/tests.zig

```diff
@@ -1,5 +1,5 @@
 comptime {
-    _ = @import("state-transition/src/lib.zig");
+    // _ = @import("state-transition/src/lib.zig");
```

Comment:
uncommenting causes 
![image](https://github.com/user-attachments/assets/31bc3177-157c-40c0-a94e-13bd48ed528f)

testing boilerplate needs fixing

Reply:
it does

Reply:
I have been investigating why this fails a bit already, the fact you broke it is going to help me further. Wait a bit before merging this PR, it's helpful for me to fix the issue in this state.

Reply:
yea uncommenting `_ = @import("state-transition/src/lib.zig");`  will show up the error. but lets merge the PR in current state itself to unblock the dev and we can address this a bit down the line I guess